android/src/main/kotlin/com/hypertrack/sdk/flutter/common/Result.kt

```diff
@@ -0,0 +1,16 @@
+package com.hypertrack.sdk.flutter.common
+
+sealed class Result<S> {
```

Comment:
There is a kotlin thing that solves it for you:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-result/

Reply:
I don't like default Kotlin Result, I think custom sealed class is better, as you can unwrap it with when for example (discussable)